lib/game/universe.go

```diff
@@ -14,23 +16,13 @@ type Universe struct {
 	newCells []uint8
 }
 
-func NewUniverse(livePopulation int) *Universe {
-	width := uint32(64)
-	height := uint32(64)
-	cells := make([]uint8, width*height)
-	newCells := make([]uint8, width*height)
-
-	for i := range cells {
-		if rand.Intn(100) < livePopulation {
-			cells[i] = Alive
-		} else {
-			cells[i] = Dead
-		}
-	}
+func NewUniverse(rows, cols uint32) *Universe {
```

Comment:
Major nitpick: I don't particularly love that we call them `rows` and `cols` here while using `height` and `width` elsewhere.

Reply:
Should it be `height` and `width` for all of the params then? I probably prefer that.

Reply:
It works either way for me - you choose 👍

Reply:
I will leave it height/width then only for `NewUniverse()` to match the fields. Force pushed update.